### PR TITLE
Grid arrowkey fix

### PR DESCRIPTION
**Pull request by mnewcomb314**
Updating the grid and treegrid roles (since treegrid depends on grid code-wise) to follow WAI-ARIA practices for the various keystrokes that determine how focus moves through a grid or treegrid

### CODE REVIEW

File: src/widgets/AppWindow.js
Comment (mnewcomb314) [inline src/widgets/AppWindow.js:22]: No longer needs to be imported due to the revised tree grid test case.  The form test case still includes testing the radio role, but that is through the RadioGroup widget.

File: src/widgets/AppWindow.js
Comment (mnewcomb314) [inline src/widgets/AppWindow.js:2100]: Refactored this test case to be closer to a real grid use case by making it like tic-tac-toe, but time wasn't available to fully implement that game, so turn swapping and victory/draw conditions aren't implemented.

File: src/widgets/TreeGrid.js
Comment (mnewcomb314) [inline src/widgets/TreeGrid.js:14]: Some of this diff is just renaming things to start with `_` to designate what should be private members from public ones.

**Review by eprouver — APPROVED** [review]

**Comment by eprouver** [comment]
I guess I don't have write access to approve - but it still LGTM

**Review by eprouver — APPROVED** [review]

**Review by LinGrin — APPROVED** [review]

**Pull request merged**
